codespell_lib/data/dictionary.txt

```diff
@@ -29269,6 +29291,9 @@ furtermore->furthermore
 furtest->furthest
 furthe->further
 furthemore->furthermore
+furthere->further
+furtheremor->furthermore
+furtheremore->furthermore
```

Comment:
How about 
furthermor->furthermore
?

Reply:
furthermor->furthermore exists already in dictionary.txt